ethpm/package.py

```diff
@@ -72,7 +72,7 @@ def get_contract_factory(self, name: ContractName, w3: Web3 = None) -> Contract:
             validate_minimal_contract_factory_data(contract_data)
         except KeyError:
             raise InsufficientAssetsError(
-                "This package has insufficient package data to generate"
+                "This package has insufficient package data to generate "
                 "a contract_factory for contract: {0}.".format(name)
```

Comment:
This is not related to the changes you made, but I wonder if you could change "contract_factory" here to "contract factory".  The underscore seems kind of random.

Reply:
Made the change - feel free to hit merge if everything else looks good